daemon/daemon.go

```diff
@@ -1353,8 +1353,14 @@ func NewDaemon() (*Daemon, error) {
 	}
 	node.SetIPv4HealthIP(health4)
 	node.SetIPv6HealthIP(health6)
-	log.Debugf("IPv4 health endpoint address: %s", node.GetIPv4HealthIP())
-	log.Debugf("IPv6 health endpoint address: %s", node.GetIPv6HealthIP())
+	localNode := node.GetLocalNode()
+	if localNode == nil {
+		log.Fatal("Error while trying to get local node")
+	}
+	localNode.UpdateHealthAddresses(node.GetIPv4HealthIP(), node.GetIPv6HealthIP())
+	localNode.OnUpdate()
```

Comment:
This needs a new function in `pkg/node/local_node.go`
```
func NotifyLocalNodeUpdated() error {
        return nodeStore.UpdateLocalKeySync(GetLocalNode())
}
```

which is then called instead of `localNode.OnUpdate()`